install/uninstall.sh

```
@@ -9,15 +16,25 @@ Warning:
 警告：
    此脚本会删除所有 pm2 托管的进程，并删除所有相关数据。
-
 Type 'Yes, do as I say!' (without quotes) below to continue:
 输入 'Yes, do as I say!' (不含引号) 继续："
 read -p "> " confirm;
 if [ "$confirm" != "Yes, do as I say!" ];then
   echo "Aborted."
   exit;
 fi
-
+if [ "$actual_user" != "$USER" ]; then
```

Shellcheck (0.11.0)

[warning] actual_user is referenced but not assigned.

(SC2154)